enrolments/models.py

```diff
@@ -68,7 +70,6 @@ class Enrolment(models.Model):
     students = ArrayField(
         models.PositiveIntegerField(),
         default=list,
-        validators=[validate_students_in_enrolment],
```

Comment:
this was previously raising an error because this was passing the list of IDs to a method that expected an enrolment object - moved this call to `validate_enrolment` (called in the model's `clean` method)